WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/AuthorSelectionAdapter.kt

```diff
@@ -122,11 +128,33 @@ class AuthorSelectionAdapter(context: Context) : BaseAdapter() {
     private class DropdownViewHolder(itemView: View) : NormalViewHolder(itemView) {
         private val text: AppCompatTextView = itemView.findViewById(R.id.post_list_author_selection_text)
 
-        override fun bind(state: AuthorFilterListItemUIState, imageManager: ImageManager, uiHelpers: UiHelpers) {
+        override fun bind(
+            state: AuthorFilterListItemUIState,
+            imageManager: ImageManager,
+            uiHelpers: UiHelpers
+        ) {
             super.bind(state, imageManager, uiHelpers)
             val context = itemView.context
+
+            // Because it's a custom popup list we need to manage colors of list items manually
+            val elevationOverlayProvider = ElevationOverlayProvider(context)
+            val appbarElevation = context.resources.getDimension(R.dimen.appbar_elevation)
+            val elevatedSurfaceColor = elevationOverlayProvider.compositeOverlayWithThemeSurfaceColorIfNeeded(
+                    appbarElevation
+            )
```

Comment:
This is quite a long dance. It was a bit difficult for me to understand at first. But that's just me coming from no experience with Dark Mode. 😅 

It feels like overkill to do this now in this PR, but do you have plans on making utility methods for this in the future? Or all usage of `ElevationOverlayProvider` is probably going to look different?

Reply:
I was thinking about using an extension for this, but I got to it a bit late -  at the moment I have a bunch of outstanding PRs with similar functionality, so I'll wait till the number reduces a bit so I can make a better decision on implementation and avoid conflicts.

Reply:
Yup! That sounds like the best approach to me. 🙂